app/models/section.rb

```diff
@@ -36,12 +36,17 @@ def self.find(manual, section_uuid, published: false)
   def initialize(manual:, uuid:, previous_edition: nil, latest_edition: nil)
     @slug_generator = SlugGenerator.new(prefix: manual.slug)
     @uuid = uuid
-    @editions = [previous_edition, latest_edition].compact.uniq
-    if @editions.empty?
-      edition = SectionEdition.new(state: "draft", version_number: 1, section_uuid: uuid)
-      @editions.push(edition)
+
+    @previous_edition = previous_edition
+    @latest_edition = latest_edition
+
+    if @previous_edition == @latest_edition
+      @previous_edition = nil
+    end
```

Comment:
I think this is related to my previous question about the use of `Array#uniq`, but why is this `if` block necessary?